tests/conftest.py

```diff
@@ -1205,6 +1205,24 @@ def pytest_generate_tests(metafunc):
     if 'enum_dut_lossy_prio' in metafunc.fixturenames:
         metafunc.parametrize("enum_dut_lossy_prio", generate_priority_lists(metafunc, 'lossy'))
 
+
+def parametrise_autoneg_tests():
+    folder = 'metadata'
+    filepath = os.path.join(folder, 'autoneg-test-params.json')
+    data = {}
+    try:
+        with open(filepath) as yf:
+            data = json.load(yf)
+    except IOError:
+        logger.warning('Cannot find a datafile for autoneg tests at {}. Run test_pretest -k test_update_testbed_metadata to create it'.format(filepath))
+        return []
```

Comment:
Suggest to re-use function `get_testbed_metadata` to read the meta data.

Reply:
`get_testbed_metadata` reads a testbed file by predefined name. To use it I'd need to put port autoneg test params there, but I intended to keep test params separately from common data present in a testbed metadata file

Reply:
`get_testbed_metadata` reads a testbed file by predefined name. To use it I'd need to put port autoneg test params there, but I intended to keep test params separately from common data present in a testbed metadata file